src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -133,6 +160,6 @@ public RobotMap() {
 		Component.xbox.setDeadZone(DefaultDriver.XBOX_MINIMUM_THRESHOLD);
 		// Main Subsystems
 		Component.alignCamera = new AligningCamera(PIDSourceType.kRate);
-		Component.mainSubsystems = new Subsystem[] { Component.chassis };
+		Component.mainSubsystems = new Subsystem[] {Component.chassis};
```

Comment:
Still not the appropriate use of subsystems. Subsystems that are being put on the dashboard, and that should by default be more that just the chassis, belong in this array. Please fix.